GOVERNANCE.md

```diff
@@ -0,0 +1,162 @@
+
+TSC meetings should have a clearly defined agenda.
+
+At any point, any TSC member may notify the TSC that they are taking a period of leave and should be considered to abstain from any signaling or voting. The TSC member must detail when they expect to return.
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
+
+A TSC member may be removed by vote from the TSC if, during a 3-month period, all of the following are true:
+
+- They do not participate in any TSC related discussions or votes
+- They do not provide any form of excuse or no excuse is known for their absence
+
+There may be other grounds for removal from the TSC, such as seriously violating the Code of Conduct.
+
+## Decision Making
+
+### Decision-making via consensus
+
+Both the Quick and Standard consensus process require the creation of an Issue in the TSC GitHub repository.
+The Issue must:
+- Include brief introductory information about the decision that needs to be made
+- Be labeled with `tsc-decision`
+- Use the provided template, unless there is a good reason not to do so
+
+#### Quick consensus process
+
+In the event a TSC member feels the need for a decision to be expedited, they may create the decision discussion, indicating that they wish to use the quick process rather than the standard consensus process. This should be done by creating a new Issue in the TSC repository.
+In addition to the requirements above, the Issue must be labeled `expedited`.
+
+While an associated Discussion is not required, if one is used, it must be clearly linked in both directions, and be labeled with `tsc-decision`.
+
+#### Quick consensus process
+
+A vote is called to see if other members agree. The vote takes place on the opening comment. This vote is different from the decision-making voting process, and only requires 3 (additional) votes in favour and none against to carry. If the vote carries, the quick process is started, otherwise the standard consensus process is started.
+
+The quick process is designed for small trivial issues where the resolution feels likely obvious, and the member who proposes the decision discussion has at least one clear resolution to present. The decision discussion owner should present the possible solution they are advocating for (in a follow up comment on the Issue. Not in the opening comment.), and ask to test for agreement. If consensus is not clearly reached, the quick process is ended and the standard consensus process is started. (Label `expedited` should be removed and `stage-1` added.)
+
+#### Standard consensus process
+
+The standard consensus process is designed for all non-trivial decisions.
+
+A decision discussion may be started by creating an Issue and Discussion in the TSC repository.
+
+In addition to the requirements above, the Issue must:
+- Be labeled with `stage-1`
```

Comment:
While reading this, I found that label confusing because it's so undescriptive. Maybe one of the following would be better.

- `tsc-decision-stage-1`
- `tsc-stage-1`
- `decision-stage-1`